lib/hardware_services/network_drivers/HilNetworkDriver.py

```diff
@@ -14,19 +14,74 @@
 from subprocess import call
 from subprocess import check_call
 
-from hardware_services.network_service import NetworkService
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", "util"))
```

Comment:
I know we can do 
`from . import echo
from .. import formats
from ..filters import equalizer`